nodes/nodes/nodes-nodes-resources-cpus.adoc

```diff
@@ -0,0 +1,20 @@
+:context: nodes-nodes-resources-cpus
+[id="nodes-nodes-resources-cpus"]
+= Allocating specific CPUs for nodes in a cluster
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+When using the xref:../../scalability_and_performance/using-cpu-manager.adoc[static CPU Manager policy], you can reserve specific CPUs for use by specific nodes in your cluster. For example, on a system with 24 CPUs, you could reserve CPUs numbered 0 - 3 for the control plane allowing the workers to use CPUs 4 - 23.
```

Comment:
s/for the control plane allowing/for the control plane, allowing

Is this an instance where we could replace `workers` with `compute nodes`?